app/src/main/res/layout/activity_view_thread.xml

```diff
@@ -6,22 +6,9 @@
     android:layout_height="match_parent"
     tools:context="com.keylesspalace.tusky.components.viewthread.ViewThreadActivity">
 
-    <com.google.android.material.appbar.AppBarLayout
-        android:id="@+id/appBar"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:elevation="@dimen/actionbar_elevation"
-        app:elevationOverlayEnabled="false">
-
-        <androidx.appcompat.widget.Toolbar
-            android:id="@+id/toolbar"
-            style="@style/Widget.AppCompat.Toolbar"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            app:contentInsetStartWithNavigation="0dp"
-            app:layout_scrollFlags="scroll|enterAlways" />
-
-    </com.google.android.material.appbar.AppBarLayout>
+    <include
+        android:id="@+id/includedToolbar"
+        layout="@layout/toolbar_basic" />
```

Comment:
For some reason, this was the only layout having its own toolbar and not including `toolbar_basic`. The toolbar scrolled with the content, unless everywhere else. I don't care either way, I just think toolbars should either scroll everywhere or nowhere. What do you prefer?